rpc/common/service.proto

```diff
@@ -41,16 +41,18 @@ message Library {
 }
 
 message Vulnerability {
-  string          vulnerability_id  = 1;
-  string          pkg_name          = 2;
-  string          installed_version = 3;
-  string          fixed_version     = 4;
-  string          title             = 5;
-  string          description       = 6;
-  Severity        severity          = 7;
-  repeated string references        = 8;
-  Layer           layer             = 10;
-  string          severity_source   = 11;
+  string                vulnerability_id  = 1;
+  string                pkg_name          = 2;
+  string                installed_version = 3;
+  string                fixed_version     = 4;
+  string                title             = 5;
+  string                description       = 6;
+  Severity              severity          = 7;
+  repeated string       references        = 8;
+  Layer                 layer             = 10;
+  string                severity_source   = 11;
+  map<string, Severity> vendor_severity   = 12;
```

Comment:
This is deprecated and will be removed. I think we don't have to add this field.